notes/lecture22.tex

```diff
@@ -0,0 +1,205 @@
+
+
+\section{Lecture 22: Non-convex constraints} 
+
+\begin{center}
+Seyed Mehran Mirramezani and Serena Yuan
```

Comment:
Please add your names to the contributors file. For instance, see https://github.com/ee227c/ee227c.github.io/pull/24/files